### PR TITLE
Descriptions added

### PR DESCRIPTION
**Pull request by fulyauluturk**

### CODE REVIEW

**Review by hasan-wajahat — APPROVED** [review]
Thanks Fulya for the well written descriptions 🙏

File: docs/synthetics/puppeteer-scripts/README.md [hunk 1]
```diff
@@ -6,16 +6,43 @@ We have added these as part of our public repository so users can improve these
...
 - [Use Sensitive Data](./use-sensitive-data.js)
+
+This scripts utilizes our secrets feature to load possible sensitive data such as usernames and passwords. We navigate to our desired page and fill up the form. For sensitive data we use the secret keys instead of the actual credentials. When this script is run our script manager will automatically replace the secret key with the actual value.
 - [Fetch and use a token](./fetch-use-token.js)
+
+This script demonstrates how to fetch a token from an authentication API, then use that token in a different request to a secure endpoint. We recommend using our [Sensitive Data](./use-sensitive-data.js) feature to store the passwords and other credentials you use for performing the authentication. 
 - [Shopping Cart](./shopping-cart.js)
+
+This scripts launch Shopify based e-commerce website, adds a book to the cart, checkout, fills the checkout form (text input and selects) and proceeds to payment.
```
Comment (otisg) [inline docs/synthetics/puppeteer-scripts/README.md:30]: This => These

File: docs/synthetics/puppeteer-scripts/README.md [hunk 2]
```diff
@@ -6,16 +6,43 @@ We have added these as part of our public repository so users can improve these
...
+- [Load  a web page](./load-web-page.js) 
+
+This is a simple example of a browser script that loads a website and takes a screenshot of a page when the navigation is complete. JPEG and PNG formats are supported   for screenshot images. Only one screenshot is allowed per script. The script gets the Puppeteer `page` object as a parameter.
 - [Navigate to a specific page](./navigate-specific-page.js)
+
+This script visits a web page, navigates to a specific page and verifies the title of the page is as expected. Node.js assert API can be used for verification. When an assertion fails, the script will fail and details of the failure can be seen in logs. The example used Puppeteer click API with specific selector to navigate to a specific page. 
+For more info on selectors check out https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors
 - [Search](./open-search-page.js)
+
+This script loads DuckDuckGo search page, enters a search term, submit the page using Enter key (simulates keypress event) and waits for the results to load. Then it extracts the first item from the search result and verifies the value of the link.
```
Comment (otisg) [inline docs/synthetics/puppeteer-scripts/README.md:18]: submit => submits

File: docs/synthetics/puppeteer-scripts/README.md [hunk 3]
```diff
@@ -6,16 +6,43 @@ We have added these as part of our public repository so users can improve these
...
 - [Login](./login-wiki.js)
+
+This script loads the web page, navigate to the login page, fill up the login details and submit the form. After login it navigates to the user profile page. Then it verifies the logged-in user name by extracting the value from the page content. You can read the values from page content using page.evaluate() API.
```
Comment (otisg) [inline docs/synthetics/puppeteer-scripts/README.md:21]: navigate => navigates

File: docs/synthetics/puppeteer-scripts/README.md [hunk 4]
```diff
@@ -6,16 +6,43 @@ We have added these as part of our public repository so users can improve these
...
 - [Use Sensitive Data](./use-sensitive-data.js)
+
+This scripts utilizes our secrets feature to load possible sensitive data such as usernames and passwords. We navigate to our desired page and fill up the form. For sensitive data we use the secret keys instead of the actual credentials. When this script is run our script manager will automatically replace the secret key with the actual value.
```
Comment (otisg) [inline docs/synthetics/puppeteer-scripts/README.md:24]: When this script is run => When this script is run,

File: docs/synthetics/puppeteer-scripts/README.md [hunk 5]
```diff
@@ -6,16 +6,43 @@ We have added these as part of our public repository so users can improve these
...
 - [Booking](./booking.js)
+
+This script launches a booking website (AirBnB) and searches for properties in a specific location and date by filling out a form. It submits the form and waits for listings to load. Then it fetches the name of the first listing and prints it. The form filling is done by driving multiple popups.
 - [Using custom metrics](./custom-metric.js)
+
+Browser monitor supports the collection of custom metrics in the script. Custom metrics can be a value from web page element, any performance metric that is not supported out of the box in Sematext Browser monitor, or any custom measurement (e.g. time to display suggestions in autocomplete) that needs custom logic. Once the measurement is added in the script, you can use Custom Chart to build charts for this custom metric. The script uses the Performance API of the Chrome browser to fetch script duration & JavaScript Heap Size and add them as custom metrics using the `context.setMetric(name, value)` API. Note that `context` is defined as the second parameter of the `testPage` function.
```
Comment (otisg) [inline docs/synthetics/puppeteer-scripts/README.md:39]: from web => from a web
and add => and adds

File: docs/synthetics/puppeteer-scripts/README.md [hunk 6]
```diff
@@ -6,16 +6,43 @@ We have added these as part of our public repository so users can improve these
...
 - [Defining custom metrics from JSON](./custom-metric-json.js)
+
+Custom metrics can also be collected from sources other than CDP. In this example, a JSON is grabbed from an API response and used to define custom metrics. Like in the example above, custom metrics defined this way can be used as data sources for charts. With custom metrics you can do a lot more than just monitor performance, such as keeping an eye out if it'll be rainy tomorrow from some weather report API and sending yourself an alert, or charting the number of users on the various services you offer over time to analyze trends. This example showcases invoking an endpoint which provides information about currency exchange rates, then defining custom metrics based on those values. These could then be used as a basis for charts, and alerts could be created if these exchange rates reach a certain point.
```
Comment (otisg) [inline docs/synthetics/puppeteer-scripts/README.md:42]: a JSON => a JSON response perhaps?

**Comment by otisg** [comment]
@fulyauluturk & @hasan-wajahat I added some comments

**Comment by fulyauluturk** [comment]
Addressed wording based on Otis' suggestions.
@hasan-wajahat I mostly copied these descriptions from SC and since the index file is not synced, we should change the wordings in SC too.

**Comment by hasan-wajahat** [comment]
Thanks @fulyauluturk I have fixed the SC copies as well.

**Pull request merged**